django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,99 @@
+from django.core import checks
+from django.db import models
+from django.db.models import NOT_PROVIDED
+from django.db.models.expressions import DatabaseDefault
+from django.utils.translation import gettext_lazy as _
+
+__all__ = ("BigSerialField", "SmallSerialField", "SerialField")
+
+
+class SerialFieldMixin:
+    db_returning = True
+
+    def __init__(self, *args, **kwargs):
+        kwargs["blank"] = True
+        super().__init__(*args, **kwargs)
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        kwargs.pop("blank")
+        return name, path, args, kwargs
+
+    def check(self, **kwargs):
+        return [
+            *super().check(**kwargs),
+            *self._check_null(),
+            *self._check_default(),
+            *self._check_database_default(),
+        ]
+
+    def _check_null(self):
+        if self.null:
+            return [
+                checks.Error(
+                    f"{self.__class__.__name__} does not accept null values.",
+                    obj=self,
+                    id="fields.E013",
+                ),
+            ]
+        else:
+            return []
```

Comment:
```suggestion
        if self.null is False:
            return []
        return [
            checks.Error(
                f"{self.__class__.__name__} does not accept null values.",
                obj=self,
                id="fields.E013",
            ),
        ]
```
(nit) let's make this the same structure as the other checks

Reply:
Okay, adjusted